File: frontend/src/app/Messages/message.ts
export class Message{
    id_message: Number;
    id_discussion: Number;
    id_sender: Number;
    message: String;
    dt: String;
    
    constructor(id_discussion: Number, id_sender: Number, message: String, dt: String){
        this.id_message = 0;
        this.id_discussion = id_discussion;
        this.id_sender = id_sender;
        this.message = message;
        this.dt = dt;
    }
}

File: LinkeDIn/backend/src/main/java/com/project/linkedin/Message/Message.java
package com.project.linkedin.Message;

import lombok.Data;

import javax.persistence.*;

@Data
@Entity
public class Message {

    private @Id @GeneratedValue(strategy = GenerationType.SEQUENCE) Long id_message;
    private Long id_discussion;
    private Long id_sender;
    private String message;
    private String dt;

    public Message() {
    }

    public Message(Long id_message, Long id_discussion, Long id_sender, String message, String dt) {

        this.id_message = id_message;
        this.id_discussion= id_discussion;
        this.id_sender= id_sender;
        this.message = message;
        this.dt= dt;

    }
}
